requirements.txt

```diff
@@ -1 +1,7 @@
 pandas==2.2.2
+numpy==2.0.1
+nltk==3.8.2
```

Comment:
@SnehaHS65 Looks like there is a problem with the `nltk` version you are using?: https://github.com/elimu-ai/ml-storybook-reading-level/actions/runs/10441098460/job/28911811221?pr=14